canonical/ubuntu.com
# WD-3695: Data docs

## Done
- Continuation of https://github.com/canonical/ubuntu.com/pull/12806
- [Create a single data documentation section](https://github.com/canonical/ubuntu.com/commit/a15c5cbdd58b0cb3caaa3ab9af899b7bbe6ed1f3)

## QA

- Go to [/data/docs](https://ubuntu-com-12969.demos.haus/data/docs) see the docs

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3695?atlOrigin=eyJpIjoiYTM1NGNiMzQ0ZmM5NDRkMDhlNjFlNDRjMzA2ZTYzYjUiLCJwIjoiaiJ9
